ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -584,6 +584,16 @@ export default class TaskGroup extends Component {
     }
   };
 
+  shouldIgnoreError(error) {
+    if (!error) {
+      return true;
+    }
+
+    // Outside Firefox CI, quite often we can have a
+    // task group without a decision task.
+    return /does not correspond to a task that exists/.test(error.message);
```

Comment:
Matching on error messages is always tricky -- if we change that error text, this would fail.  Does web-server forward the error code from the backend?  If not, then this is fine (a regression wouldn't be the end of the world).

Reply:
Yes. I added a patch :)

Reply:
Awesome, much better..